methods.tex

```diff
@@ -427,8 +426,7 @@ \section{Reactor Specifications}
 After each 18 month cycle, one-third of the 
 core (77 assemblies) discharges. Refueling
 is assumed to take 2 months to complete, during which the reactor
-is shut down. This value is acquired by averaging the 
-historical refueling outage. The specifications are defined in 
+is shut down. The specifications are defined in 
```

Comment:
This edit has resulted in a strange sentence "The specifications are defined in Table X details the reactor specifications in this simulation." I think you mean "The specifications are defined in Table X which details the reactor specifications in this simulation."